features/eolearn/features/bands_extraction.py

```diff
@@ -47,17 +47,21 @@ def map_method(self, feature):
 class NormalizedDifferenceIndexTask(MapFeatureTask):
     """ The task calculates a Normalized Difference Index (NDI) between two bands A and B as:
 
-        :math:`NDI = \\dfrac{A-B}{A+B}`
+        :math:`NDI = \\dfrac{A-B+c}{A+B+c}`,
+
+        where c is provided as the *constant* argument.
     """
 
-    def __init__(self, input_feature, output_feature, bands, undefined_value=np.nan):
+    def __init__(self, input_feature, output_feature, bands, constant=0, undefined_value=np.nan):
```

Comment:
Would it make sense rename `constant` to `acorvi_constant`? This way users might get some context what this constant represents.

Reply:
Done.